tools/proteomiqon_proteininference/proteomiqon_proteininference.xml

```diff
@@ -0,0 +1,267 @@
+    <inputs>
+        <param name="peptideDB" type="data" format="sqlite" label="Peptide database" help="Specify the peptide data base."/>
+        <conditional name="PSMInputModeCond">
+            <param name="PSMInputMode" type="select" label="PSM file input mode">
+                <option value="single">Single File</option>
+                <option value="multi">Multiple Files</option>
+            </param>
+            <when value="single">
+                <param name="scoredPSMs" type="data" format="tabular" label="Scored PSM file" help="Specify peptide identifications." />
+            </when>
+            <when value="multi">
+                <param name="scoredPSMs" type="data" format="tabular" label="Scored PSM files" help="Specify list of peptide identifications." multiple="true"/>
+            </when>
+        </conditional>
+        <param name="ProteinIdentifierRegex" type="text" value="id" label="Protein identifier regex pattern" help="Fasta headers do often contain additional information in addition to your protein identifier, by specifying a regex pattern the tool can extract the protein IDs. If you fasta headers are already cleaned you can leave this field empty.">
+            <sanitizer sanitize="false" />
+        </param>
+        <param name="Protein" type="select" label="Protein" help="Specify how protein groups are created. For details please refer to the description below.">
+            <option value="Minimal">Minimal</option>
+            <option value="Maximal" selected="true">Maximal</option>
+        </param>
+        <param name="Peptide" type="select" label="Peptide" help="Specify how peptides are used to infer protein groups. For details please refer to the description below.">
+            <option value="Minimal" selected="true">Minimal</option>
+            <option value="Maximal">Maximal</option>
+            <option value="MaximalInverse">MaximalInverse</option>
+        </param>
+        <param name="GroupFiles" type="boolean" checked="true" label="Groupe files" help="If checked, protein inference is carried out using peptide information from all files simultaneously."/>
+        <conditional name="GetQValueCond">
+            <param name="GetQValue" type="select" label="Q-Value method" help="Specify if and how q-value calculation should be carried out.">
+                <option value="Storey" selected="true">Storey</option>
+                <option value="LogisticRegression">Logistic Regression</option>
+                <option value="NoQValue">NoQValue</option>
+            </param>
+            <when value="LogisticRegression">
+                <param name="LogisticRegressionType" type="select" label="Logistic regression type" help="">
+                    <option value="Conservative">Conservative</option>
+                    <option value="MAYU" selected="true">MAYU</option>
+                    <option value="DecoyTargetRatio">DecoyTargetRatio</option>
+                </param>
+            </when>
```

Comment:
the other "whens" are missing here

Reply:
mh, why is out linter not complaining, wired...

Reply:
These are union cases, the other whens are not needed, it's just the LogisticRegression that follows up with extra possibilities.

Reply:
That is fine, just from a formal point of view we need `<when value="Storey"/>` and  `<when value="NoQValue"/>`

Reply:
Understood!